discovery-infra/test_infra/assisted_service_api.py

```diff
@@ -253,6 +253,11 @@ def set_pull_secret(self, cluster_id: str, pull_secret: str) -> models.cluster.C
         update_params = models.ClusterUpdateParams(pull_secret=pull_secret)
         return self.update_cluster(cluster_id=cluster_id, update_params=update_params)
 
+    def set_labels(self, cluster_id: str, hosts_with_labels) -> models.cluster.Cluster:
+        log.info("Setting labels for hosts %s in cluster %s", hosts_with_labels, cluster_id)
+        update_params = models.ClusterUpdateParams(host_labels=hosts_with_labels)
+        return self.update_cluster(cluster_id=cluster_id, update_params=update_params)
```

Comment:
should be v2_update_cluster

Reply:
This method is called on line 263. Here we call local method.